rules/falco_rules.yaml

```diff
@@ -2415,6 +2415,18 @@
     WARNING
   tag: [process, mitre_defense_evation]
 
+# This rule is deprecated and will/should never be triggered. Keep it here for backport compatibility.
```

Comment:
Can you mention that Delete or rename shell history is the preferred rule now?

Reply:
Done. Thanks!